rfcs/062-knowledge-graph/README.md

```diff
@@ -0,0 +1,234 @@
+
+* A graph can capture complex relationships between diverse sources of information with multiple types of edges and properties. This can include both structured and unstructured data.
+* A graph can evolve as new data becomes available or obsolete. One can add/remove nodes and edges without restructuring the entire model.
+* Graph databases are optimised for executing complex queries and traversal of relationships, revealing new paths and connections between concepts which can be utilised in search
+* The graph structure enables mathematical functions and machine learning, such as finding shortest paths or patterns, clustering, entity disambiguation, predicting missing attributes, recommending new concepts, and more.
+* Graphs can be visualised, making it possible to show patterns, outliers, or clusters within the data.
+
+A prototype graph was developed for the Collection in the past, linking existing concepts to external ontologies such as Wikidata (discussed in detail in the next section). There is some information available online from other institutions which have developed graphs for collection/museum content, namely the Science Museum's heritage connector [1] and a graph created by the Chinese Palace Museum [2]. The Science Museum's approach also seems to be primarily focused around linking records to Wikidata, with an additional component involving named entity recognition (NER) and entity linking. The Chinese Palace Museum's approach seems slightly different and follows a more traditional knowledge graph construction via NER and relation extraction from item descriptions. It is also worth noting that the machine learning team at Wellcome has successfully built a graph in Neo4j containing more than one billion historic citation relationships and other metadata relating to academic entities (Wellcome Academic Graph), however this is only available internally. 
+
+## Overview of knowledge graph v1 (archived)
+
+Prototype pipelines were developed in 2022 to construct a [Neo4j](https://neo4j.com/) graph for the Collection. The source code (written in Python) can be viewed in the archived [knowledge-graph](https://github.com/wellcomecollection/knowledge-graph) repository on github. The code is not well-documented and does not include tests. Specific pipline logic can only be deduced from reading the code itself. The purpose of this graph was to enrich existing concepts with data from Library of Congress, Medical Subject Headings (MeSH), and Wikidata. These sources reference each other via fields such as "closely matching" identifiers, and this was used to create edges between concepts in the graph. This graph was only used for enrichment of existing concepts, not to extract/infer new concepts. Its main purpose was to surface this data in an elasticsearch index. As far as it is possible to tell from the repository code, the graph database itself was not associated with any dedicated infrastructure and only created locally.
```

Comment:
```suggestion
Prototype pipelines were developed in 2022 to construct a [Neo4j](https://neo4j.com/) graph for the Collection. The source code (written in Python) can be viewed in the archived [knowledge-graph](https://github.com/wellcomecollection/knowledge-graph) repository on GitHub. The code is not well-documented and does not include tests. Specific pipeline logic can only be deduced from reading the code itself. The purpose of this graph was to enrich existing concepts with data from Library of Congress, Medical Subject Headings (MeSH), and Wikidata. These sources reference each other via fields such as "closely matching" identifiers, and this was used to create edges between concepts in the graph. This graph was only used for enrichment of existing concepts, not to extract/infer new concepts. Its main purpose was to surface this data in an Elasticsearch index. As far as it is possible to tell from the repository code, the graph database itself was not associated with any dedicated infrastructure and only created locally.
```

typo to "pipeline" and capitalised some tool names while I was at it but it's very much a np; enjoying my read so far 👀 😊